src/sail_lean_backend/pretty_print_lean.ml

```diff
@@ -10,31 +10,32 @@ open PPrint
 open Pretty_print_common
 
 type context = {
+  env : Type_check.env;
```

Comment:
I think it would be useful to say which environment this is, especially as the final global environment won't contain local bindings that may be necessary when translating expressions.

Reply:
You mean in a comment?

Reply:
Yes, just a comment.

Reply:
Done!